### PR TITLE
move version and update user agent with client name and version

### PR DESCRIPTION
**Pull request by cp16net**

### CODE REVIEW

File: planet/api/dispatch.py [hunk 1]
```diff
@@ -92,6 +93,10 @@ def rebuild_auth(self, prepared_request, response):
                     )
 
 
+def _get_user_agent():
+    return 'planet-python-client/%s' % __version__
```
Comment (ischneider) [inline planet/api/dispatch.py]: to reduce potential confusion, can we just use the project name?

**Comment by ischneider** [comment]
Looks nice and thanks for the header debug addition. Minor comment re agent name.

File: planet/api/dispatch.py (hunk 1 above)
Comment (cp16net) [reply planet/api/dispatch.py]: oh man i must have been dreaming and thought thats what the project name was...

Comment (cp16net) [reply planet/api/dispatch.py]: fixed

**Pull request merged**